text/02-FirstGame.md

```diff
@@ -97,151 +111,94 @@ To play the game inside Blender, open the file game.blend. This file is only a p
 
 To follow the progress of the instruction steps, we have other folders. Copy the whole folder onto your computer to work from there. These are the folders we will be using:
 
-**\** Book\Chapter02\game\_my **\** -The semi-empty folder structure to be filled as you advance in the chapter.
+- **Book/Chapter02/game\_my** -The semi-empty folder structure to be filled as you advance in the chapter.
 
-\Book\Chapter02\game\_progress\-The same folder structure but filled with files of different snapshots. Every file is named after the original name plus a progress number-for example, game.1.blend, game.1.blend, //assets/shark.8.blend. To use them, you need to rename the file to the original name and copy to the right folder in "game\_my" folder.
+- **Book/Chapter02/game\_progress** -The same folder structure but filled with files of different snapshots. Every file is named after the original name plus a progress number-for example, game.1.blend, game.1.blend, //assets/shark.8.blend. To use them, you need to rename the file to the original name and copy to the right folder in "game\_my" folder.
 
-\Book\Chapter02\game\_final\-The final as of the end of this exercise; use for reference.
+- **Book/Chapter02/game\_final** -The final as of the end of this exercise; use for reference.
 
-\Book\Chaper02\references\-Files to support the making of the game.
+- **Book/Chaper02/references** -Files to support the making of the game.
 
 For the rest of the chapter, we will refer to the files from your //game\_my/ top folder.
 
+
 ## Modeling
```

Comment:
Modeling need more clarification and images. There are some gaps between modeling steps.